tests/introspection/tests.py

```diff
@@ -171,8 +171,8 @@ def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
         if connection.features.can_introspect_index_type:
-            self.assertIn('type', indexes['reporter_id'])
-            indexes['reporter_id'].pop('type')
+            index_type = indexes['reporter_id'].pop('type', None)
+            self.assertEqual(index_type, 'btree')
```

Comment:
That would definitely be better than arbitrarily deciding the name.

That way, ideally, we might have to import all indexes into the introspection files and then we can store the mapping in the index class or the introspection file.
Or we could hard code the suffixes in the introspection file. This should be feasible given that suffixes rarely going to change for any index.

I like the second way more.